src/actions/index.js

```diff
@@ -0,0 +1,20 @@
+let nextBookId = 0;
+
+export const createBook = ({ title, category }) => ({
+  type: 'CREATE_BOOK',
+  book: {
+    id: nextBookId++,
```

Comment:
In response to your question:

> The project instructions are confusing at times. In this case I don't understand how I am supposed to use `Math.random()` to have a unique value each time...
> 
> +1 Nice idea about the refactor. The reducer looks much better now indeed: `{ ...action.book }`
> Thanks

You are right! I personally don't like to use `Math.random()` for this as for you to be able to do that you need to add more tweaks to the simple Math.random.. I highly recommend you use [`uuid`](https://github.com/kelektiv/node-uuid#readme) for this.. It's easy to use. Just import it here and run `uuid()` whenever you want to generate an ID.